cmd/spruce/main_test.go

```diff
@@ -250,6 +250,53 @@ map:
 			So(stderr, ShouldEqual, "")
 		})
 
+		Convey("Should not evaluate spruce logic appart from (( prune )) when --no-eval", func() {
```

Comment:
Can this test case name be updated?  It looks like the test is ensuring that the prune operator and other operators are not evaluated